web/styles/settings.css

```diff
@@ -633,6 +633,12 @@ input[type="checkbox"] {
             font-weight: bold;
             word-break: break-all;
         }
+
+        .zulip-icon {
+            font-size: 0.93em;
```

Comment:
@AMANVISHWAKARMA27 can you comment on why you want to use 0.93em here?

Reply:
It doesn't have any special reason. I just looked around other @ icons like that in the sidebar and tried hit and trial to get a perfect size. 
If there can be a better approach, please tell me.

Reply:
I believe I should keep the font-size at 0.8em, similar to the "@" icon next to the unread message count in the sidebar.

Reply:
That is the right call. Should we `var(--base-font-size-px)` instead?

Reply:
Sure.

Reply:
So, as per your suggestions, changed `font-size: 0.8em` to `font-size: calc(0.8*var(--base-font-size-px))`.
I hope the usecase is correct.